web/components/editor.tsx

```diff
@@ -178,7 +183,23 @@ export function TextEditor(props: {
                   setOpen={setIframeOpen}
                 />
                 <CodeIcon className="h-5 w-5" aria-hidden="true" />
-                <span className="sr-only">Embed an iframe</span>
+              </button>
+            </div>
+            <div className="tooltip flex items-center" data-tip="Add market">
+              <button
+                type="button"
+                onClick={() => setMarketOpen(true)}
+                className="-m-2.5 flex h-10 w-10 items-center justify-center rounded-full text-gray-400 hover:text-gray-500"
+              >
+                <MarketModal
```

Comment:
I'm not sure why it brings up the 'unsaved changes warning' but if we can't figure out how to officially let the window know it's no big deal we could toggle it off/on when interacting with this button. Check out `use-warn-unsaved-changes.ts` to see where it's toggled on.

Reply:
I think an actual redirect happens. It should warn unsaved changes in that case, when adding stuff to the post